core/templates/dev/head/appSpec.js

```diff
@@ -137,9 +137,9 @@ describe('Datetime Formatter', function() {
     it('should show the date even for a datetime occurring today', function() {
       // In any timezone, 10 minutes before xx:45:00 should still fall within
       // the same date as xx:45:00 in getLocaleDateString(). The date should
-      // always be shown as '11/21/2014'
-      expect(df.getLocaleDateString(
-        NOW_MILLIS - 10 * 60 * 1000)).toBe('11/21/2014');
+      // always be shown as '11/21/2014' or '21/11/2014'.
+      expect(['11/21/2014', '21/11/2014']).toContain(df.getLocaleDateString(
```

Comment:
Ah, good question. I've updated oppia/oppia/i18n-issue-103, so if you merge mit0110/oppia/i18n-issue-103 with that branch, this should be gone now. (It is probably possible to merge 'develop' in directly, too, but it's conceptually more complicated and, although I think we'll get eventual consistency, I'm not entirely sure what will happen.)